scripts/zkai_attention_kv_two_head_fused_softmax_table_native_gate.py

```
@@ -0,0 +1,1049 @@
+    )
+    if sidecar_envelope.get("source_input") != source_input:
+        raise AttentionKvTwoHeadFusedSoftmaxTableGateError("sidecar source input split-brain drift")
+    parse_stark_proof(
+        sidecar_envelope.get("proof"),
+        SIDECAR_PROOF_SIZE_BYTES,
+        "sidecar",
+        expected_commitments=SIDECAR_PROOF_COMMITMENTS,
+    )
+    lookup_summary = sidecar_envelope.get("lookup_summary")
+    if not isinstance(lookup_summary, dict):
+        raise AttentionKvTwoHeadFusedSoftmaxTableGateError("sidecar lookup summary missing")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)